### PR TITLE
[new] script

### PR DESCRIPTION
**Pull request by GustavoRSSilva**

### CODE REVIEW

File: contracts/AppCoins.sol [hunk 1]
```diff
@@ -17,6 +17,8 @@ contract AppCoins is ERC20Interface{
     // Public variables of the token
```
Comment (DioPires) [inline contracts/AppCoins.sol]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 48 passing
```
#### Test coverage: 
78.47 %

Comment (DioPires) [inline contracts/AppCoins.sol]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 48 passing
```
#### Test coverage: 
78.47 %

Comment (DioPires) [inline contracts/AppCoins.sol]: #### Tests:
```diff 
- 1 failing
```
```diff 
+ 48 passing
```
#### Test coverage: 
78.47 %

**Review by tcrosado — APPROVED** [review]

**Pull request merged**